examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/Basic.cs

```diff
@@ -30,9 +31,9 @@ public void Run(string[] args)
                 .Config("spark.some.config.option", "some-value")
                 .GetOrCreate();
 
-            // Need to explicitly specify the schema since pickling vs. arrow formatting
-            // will return different types. Pickling will turn longs into ints if the values fit.
-            // Same as the "age INT, name STRING" DDL-format string.
+            // Need to explicitly specify the schema since pickling vs.arrow formatting
+            // will return different types.Pickling will turn longs into ints if the values fit.
+            // Same as the "age INT, name STRING" DDL - format string.
```

Comment:
Any reason to remove and add spaces here?

Reply:
Good catch, will remove these